valhalla/baldr/double_bucket_queue.h

```diff
@@ -147,18 +179,18 @@ class DoubleBucketQueue final {
     }
 
     // Return label from lowest non-empty bucket
-    uint32_t label = currentbucket_->back();
+    const uint32_t label = currentbucket_->back();
     currentbucket_->pop_back();
     return label;
   }
 
 private:
-  float bucketrange_; // Total range of costs in lower level buckets
-  float bucketsize_;  // Bucket size (range of costs in same bucket)
-  float inv_;         // 1/bucketsize (so we can avoid division)
-  double mincost_;    // Minimum cost within the low level buckets
-  float maxcost_;     // Above this goes into overflow bucket
-  float currentcost_; // Current cost
+  float bucketrange_{}; // Total range of costs in lower level buckets
```

Comment:
Are these brackets needed since default initialization will be 0 anyway?

Reply:
I added default c-tor, so I need to initialize all these values to 0